doc/source/reference/api/grn_table.rst

```diff
@@ -139,11 +139,18 @@ Reference
 
 .. c:type:: grn_table_sort_key
 
-   TODO...
+   .. c:member:: grn_obj* key
+   .. c:member:: grn_table_sort_flags flag
+   .. c:member:: int offset
 
 .. c:type:: grn_table_sort_flags
 
-   TODO...
+   A typedef of unsigned char
```

Comment:
This is needless information.
It breaks abstration.

Instead, we need describe when we need to use this flags.